tgui/packages/tgui/interfaces/NtosNetDownloader.tsx

```diff
@@ -1,14 +1,41 @@
+import { BooleanLike } from 'common/react';
 import { scale, toFixed } from 'common/math';
 import { useBackend, useLocalState } from '../backend';
-import { Box, Button, Stack, Icon, LabeledList, NoticeBox, ProgressBar, Section, Tabs } from '../components';
+import { createSearch } from 'common/string';
+import { Box, Button, Stack, Icon, Input, LabeledList, NoticeBox, ProgressBar, Section, Tabs } from '../components';
 import { flow } from 'common/fp';
 import { filter, sortBy } from 'common/collections';
 import { NtosWindow } from '../layouts';
 
+type NetDownloaderData = {
```

Comment:
I'd like to avoid this pattern for backend data lest our types start to resemble c#. Just `Data` is fine